src/actions/getMyRecipes.ts

```diff
@@ -0,0 +1,28 @@
+import { prisma } from "../lib/prisma";
+import { getAuthenticatedUser } from "./getAuthenticatedUser";
+
+const getMyRecipes = async () => {
+  const user = await getAuthenticatedUser();
+
+  if (!user) {
+    return null;
+  }
```

Comment:
```suggestion
  if (!user) {
    throw new Error("認証に失敗しました");
  }
```

user取得できないないときはエラー返す法が良いかと！